source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -573,6 +573,16 @@ Octopus.Client
     void SetParameter(String, IDictionary<String, String>)
   }
 }
+Octopus.Client.AutomationEnvironments
+{
+  class AutomationEnvironmentProvider

```

Comment:
This is only exposing the information retrieval side of things - cant modify anything internal.
This is needed for the LogExtensions in Octopus.Cli